services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/internal/operations/BatchWriteItemOperationTest.java

```diff
@@ -179,6 +179,39 @@ public void generateRequest_multipleTables_mixedCommands_usingShortcutForm() {
                                                       deleteRequest(FAKESORT_ITEM_MAPS.get(2))));
     }
 
+    @Test
+    public void generateRequest_multipleTables_mixedCommands_usingKeyItemForm() {
+        BatchWriteItemEnhancedRequest batchWriteItemEnhancedRequest =
+            BatchWriteItemEnhancedRequest.builder()
+                                         .writeBatches(
+                                             WriteBatch.builder(FakeItem.class)
+                                                       .mappedTableResource(fakeItemMappedTable)
+                                                       .addPutItem(FAKE_ITEMS.get(0))
```

Comment:
The overall effect when you mix-n-match using the new format is one of pleasing uniformity, I think it enhances the usability.